tox.ini

```diff
@@ -173,3 +148,22 @@ commands =
         --cache-dir="{toxworkdir}/mypy_cache"  \
         {tty:--pretty:}                        \
         {posargs:src}
+
+;
+; Create sdist and wheel packages and run basic tests on them.
```

Comment:
```suggestion
; Create sdist, extract wheel, and run basic tests on them.
```

Reply:
Well, even if the wheel is extracted from tox... the overall result of this env is that a wheel is created :)... how it is created, is low-level technical detail :)